src/main/java/org/opensrp/connector/dhis2/DHIS2ImportLocationsStatusService.java

```diff
@@ -0,0 +1,43 @@
+package org.opensrp.connector.dhis2;
+
+import org.opensrp.domain.AppStateToken;
+import org.opensrp.repository.AppStateTokensRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class DHIS2ImportLocationsStatusService {
+
+	private final AppStateTokensRepository allAppStateTokens;
+
+	public DHIS2ImportLocationsStatusService(AppStateTokensRepository allAppStateTokens) {
```

Comment:
Missing Autowired annotation on the setter or the field

Reply:
Added